### ISSUE TITLE
Fetch is blocked by CORS policy

### ISSUE THREAD

**Issue by GELight** [larrybolt/online-ics-feed-viewer#15]
Hi,

We try to use your calendar feed for a customer of us.
This is his link to the ical data: https://www.kuk-ferienwohnungen.de/system/modules/fewomanager/FewoIcal.php?id=1192620e3176ff07502d

Generated link is:
```
https://larrybolt.github.io/online-ics-feed-viewer/#feed=https%3A//www.kuk-ferienwohnungen.de/system/modules/fewomanager/FewoIcal.php%3Fid%3D1192620e3176ff07502d&cors=true&title=My%20Feed&hideinput=false
```

Upload is possible, but automatic retrieval (fetch) is blocked by CORS policy.

```
Access to XMLHttpRequest at 'https://www.kuk-ferienwohnungen.de/system/modules/fewomanager/FewoIcal.php?id=1192620e3176ff07502d' from origin 'https://larrybolt.github.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

We try to use your calendar as iframe...
Any ideas?

Best regards,
Mario

**Comment by lucasdepetrisd** [comment]
I'm having the same issue. Did you manage to solve it?

**Comment by lucasmendes** [comment]
Same issue for me. Lots of CORS errors in Console.

**Comment by nagyv** [comment]
TL; DR: Use https://calendar-sharing.nagyv.com/  No data is saved.

The issue is that this frontend-only app needs a backend service to do the CORS call. It uses https://larrybolt-cors-anywhere.herokuapp.com/, but that app is not available any more. A similar app could be https://cors-anywhere.herokuapp.com/corsdemo, that has a neat description why it might not fit many use cases: https://github.com/Rob--W/cors-anywhere/issues/301
